packages/x-components/src/tailwind/plugin-options.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  theme: {
+    colors: {
+      neutral: {
+        25: '#f2f2f2',
+        50: '#b3b3b3',
+        75: '#595959'
+      },
+      primary: {
+        50: '#36515b'
+      },
+      secondary: {
+        50: '#667981'
+      },
+      accent: {
+        50: '#b90276'
+      },
+      success: {
+        50: '#00705c'
+      },
+      error: {
+        50: '#e11f26'
+      }
+    },
+    spacing: {
+      72: 72,
+      74: 74,
+      120: 120,
+      150: 150,
+      202: 202,
+      282: 282
+    },
+    fontFamily: {
+      primary: ['Montserrat', 'sans-serif'],
+      secondary: ['Lora', 'sans-serif'],
+      auxiliary: 'Space Mono',
+      special: '',
+      icons: ''
+    },
+    fontSize: {
+      lg: '18px'
+    },
+    fontWeight: {
+      light: 300
+    },
+    lineHeight: {
+      legacy: {
+        sm: '16px',
+        md: '24px',
+        lg: '32px'
+      }
```

Comment:
There's a piece of code that is using the tailwind class `leading`, that uses `lineHeight`. I went this route instead of replacing the plugin's original values, but the other option is changing the element that uses `leading` instead.